molgenis-data-annotators/src/main/java/org/molgenis/data/annotation/entity/impl/GavinAnnotator.java

```diff
@@ -0,0 +1,351 @@
+								+ mafReason);
+			}
+		}
+
+		// CADD score based classification, calibrated
+		if (caddScaled != null)
+		{
+			if ((category.equals(Category.C1) || category.equals(Category.C2)))
+			{
+				if (caddScaled > meanPathogenicCADDScore)
+				{
+					return new Judgment(Judgment.Classification.Pathognic, Method.calibrated,
+							"Variant CADD score of " + caddScaled + " is greater than the mean pathogenic score of "
+									+ meanPathogenicCADDScore
+									+ " in a gene for which CADD scores are informative. Also, " + mafReason);
+				}
+				else if (caddScaled < meanPathogenicCADDScore)
+				{
+					return new Judgment(Judgment.Classification.Benign, Method.calibrated,
+							"Variant CADD score of " + caddScaled + " is lesser than the mean population score of "
+									+ meanPathogenicCADDScore
+									+ " in a gene for which CADD scores are informative, although " + mafReason);
+				}
+			}
+			else if ((category.equals(Category.C3) || category.equals(Category.C4) || category.equals(Category.C5)))
+			{
+				if (caddScaled > spec95thPerCADDThreshold)
+				{
+					return new Judgment(Judgment.Classification.Pathognic, Method.calibrated,
+							"Variant CADD score of " + caddScaled + " is greater than the 95% specificity threhold of "
+									+ spec95thPerCADDThreshold + " for this gene. Also, " + mafReason);
+				}
+				else if (caddScaled < spec95thPerCADDThreshold)
+				{
+					return new Judgment(Judgment.Classification.Benign, Method.calibrated,
+							"Variant CADD score of " + caddScaled + " is lesser than the 95% sensitivity threhold of "
+									+ spec95thPerCADDThreshold + " for this gene, although " + mafReason);
+				}
+			}
+		}
+
+		// if everything so far has failed, we can still fall back to the naive method
+		return genomewideClassifyVariant(impact, caddScaled, exacMAF, gene);
+	}
+
+	private Map<String, Double> toMap(String alternatives, String annotations)
+	{
+		if (annotations == null) annotations = "";
+		String[] altArray = alternatives.split(",");
+		String[] annotationsArray = annotations.split(",");
+
+		Map<String, Double> result = new HashMap<>();
+		if (altArray.length == annotationsArray.length)
+		{
+			for (int i = 0; i < altArray.length; i++)
+			{
+				Double value = null;
+				if (StringUtils.isNotEmpty(annotationsArray[i]))
+				{
+					value = Double.parseDouble(annotationsArray[i]);
+				}
+				result.put(altArray[i], value);
+			}
+		}
+		else if (StringUtils.isEmpty(annotations))
+		{
+			for (int i = 0; i < altArray.length; i++)
+			{
+				result.put(altArray[i], null);
+			}
+		}
+		else
+		{
+			throw new MolgenisDataException(VcfRepository.ALT + " differs in length from the provided annotations.");
+		}
+		return result;
+	}
+
+	public Judgment genomewideClassifyVariant(Impact impact, Double caddScaled, Double exacMAF, String gene)
+	{
+
+		exacMAF = exacMAF != null ? exacMAF : 0;
+
+		if (exacMAF != null && exacMAF > MAF_THRESHOLD)
```

Comment:
you can remove "exacMAF != null" from "if (exacMAF != null && exacMAF > MAF_THRESHOLD)"
it is never null